jetty-documentation/src/main/asciidoc/development/maven/jetty-maven-plugin.adoc

```diff
@@ -324,7 +324,7 @@ Defaults to "true".
 Controls whether any overlaid wars are added before or after the original base resource(s) of the webapp.
 See the section on link:#using-overlaid-wars[overlaid wars] for more information.
 containerIncludeJarPattern;;
-Defaults to `.*/javax.servlet-[^/]*\.jar$|.*/servlet-api-[^/]*\.jar$|.*javax.servlet.jsp.jstl-[^/]*\.jar|.*taglibs-standard-impl-.*\.jar`.
+Defaults to `.*/javax.servlet-[^/]*\.jar$|.*/[^/]*servlet-api-[^/]*\.jar$|.*javax.servlet.jsp.jstl-[^/]*\.jar|.*taglibs-standard-impl-.*\.jar`.
```

Comment:
I still haven't seen an explanation of why the servlet-api pattern has had an extra [^/] added? Why is this necessary, and if it is, shouldn't it be applied to all other jars as well?

Reply:
I thought I reverted all of those.
Hmm, that's the documentation too.

Reply:
The old pattern `.*/servlet-api-[^/]*\.jar$` would not match against `jetty-servlet-api-4.0.2.jar`

Reply:
and if you do a mid-path search, you want to make sure that you hit on `jetty-home/lib/jetty-servlet-api-4.0.2.jar` and not something like `jetty-home/lib/servlet-api-4.0/schemas.jar`

Reply:
Well, the pattern should be  .*/jetty-servlet-api-[^/]*\.jar$ then.

Reply:
this was reverted.